File: go/day40.go
package main

import (
	"fmt"
	"time"
)

func main() {
	var nyLocation, _  = time.LoadLocation("America/New_York");
	var nyTime = time.Now().In(nyLocation);

	fmt.Print(
		"\n\nCurrent Date/Time in NYC: " +
		nyTime.Format("Mon Jan 02 2006") +
		" " +
		nyTime.Format("15:04:05") +
		"\n",
	)
}

File: cSharp/day40.cs

using System;


DateTime dateUtc = DateTime.UtcNow;

TimeZoneInfo nyTimezone = TimeZoneInfo
    .FindSystemTimeZoneById("America/New_York");

DateTime nyTime = TimeZoneInfo
    .ConvertTimeFromUtc(dateUtc, nyTimezone);


Console.WriteLine(
    "Time in NYC: "+
    nyTime.ToString("MMM dd yyyy HH:mm:ss tt")
);

